### PR TITLE
rework ovirt-orb

### PR DESCRIPTION
**Pull request by dankenigsberg**
The former version had too many headers relative to the content within. My version condenses the content and makes using it more straightforward.

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @dankeningsberg

I would like to add a paragraph on how orb is built. @lveyde, where is the code that creates the tarball?

This pull request needs review by: @sandrobonazzola @lveyde

### CODE REVIEW

File: source/documentation/ovirt-orb/index.html.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 category: documentation
-layout: toc
 ---
 
 # oVirt Orb
 
-oVirt Orb is a project that allows anyone to easily take an oVirt for a ride, test and play with it.
-All without the need to manually install all the components.
-It's based on the Lago framework and runs the whole environment on a single machine by creating a number of VMs, using a concept of nested virtualization.
+`oVirt Orb` lets you try out oVirt on your own laptop. You can test it and play with it, all without the need to manually install all the components or use multiple host or a storage server. `oVirt Orb` is based on the Lago framework and runs the whole oVirt environment on a single machine by creating a number of VMs. VMs that play as hypervisors can run nested VMs.
```
Comment (mmirecki) [inline source/documentation/ovirt-orb/index.html.md]: host -> hosts
" use multiple hosts or"

File: source/documentation/ovirt-orb/index.html.md [hunk 2]
```diff
@@ -30,63 +29,52 @@ It's based on the Lago framework and runs the whole environment on a single mach
...
+    https://192.168.202.4
+    Username: admin
+    Password: 123
 
-    lago ovirt start
-
-On the screen you should see oVirt engine's IP, username, and password.
-
-You can enter to the web UI by entering the engine's IP in your browser.
+You can enter to oVirt Engine's web UI by browsing to the supplied URL. You may need to add an exception for your browser to trust the self-signed certificate created by Lago.
```
Comment (mmirecki) [inline source/documentation/ovirt-orb/index.html.md]: delete 'to'

You can enter to oVirt Engine's web => You can enter oVirt Engine's web

**Comment by rollandf** [comment]
Also maybe we can add a section describing what kind of setup we will get in this demo:
- Hosted engine?
- How many hosts?
- Storage types?
- Networking?
- Already defined VMs?

File: source/documentation/ovirt-orb/index.html.md [hunk 3]
```diff
@@ -30,63 +36,59 @@ It's based on the Lago framework and runs the whole environment on a single mach
...
+URL. You may need to add an exception for your browser to trust
+the self-signed certificate created by Lago.
 
-On the screen you should see oVirt engine's IP, username, and password.
-
-You can enter to the web UI by entering the engine's IP in your browser.
+As of version 4.2.2, the environment starts up three VMs; one
+for Engine and storage, and two others for virtual hypervisors,
+interconnected via several virtual networks. The precise make
+of the environment may change in the future.
```
Comment (dankenigsberg) [inline source/documentation/ovirt-orb/index.html.md]: @rollandf I prefer not to delve too deeply into the env implementation - it smells like something that can change quicker than this doc page.

Comment (rollandf) [reply source/documentation/ovirt-orb/index.html.md]: OK

**Comment by dankenigsberg** [comment]
@sandrobonazzola I do not understand how people work with github :-(
I've updated my branch to d5ede794f1b2333106bc89de311b0cb8ca7c1595 which do not touch the submodule (now idea how that has happened, too) but I see no note of it in the conversation tab.

anyway, please re-review.

**Pull request merged**